R/annotateComplexFeatures.R

```diff
@@ -66,7 +60,7 @@
 #'          }
 #'        }
 
```

Comment:
I think we can live without an example here at this moment because I hope to integrate this function somewhere else later anyways.